src/service/api/index.ts

```diff
@@ -20,4 +22,17 @@ api.interceptors.request.use((config: AxiosRequestConfig) => {
   return config;
 });
 
+api.interceptors.response.use(
+  (response) => {
+    return response;
+  },
+  (error) => {
+    if (error.response.status === 401) {
+      useClearStorage();
+      window.location.href = SCREENS.LOGIN;
+    }
```

Comment:
Tentei fazer login passando senha errada e a tela ficava recarregando, o que atrapalha na experiência já que a msg de login e/ou senha errados não é mostrada. O ideal seria você só redirecionar o login se o usuário estiver em uma rota em que precisa estar logado.

Reply:
Uma opção é no arquivo `src/utils/screens.ts` você exportar uma lista de telas em que não precisa estar logado, daí aqui nesse interceptor vc verifica se o nome da rota não está nessa lista para pode forçar o login novamente.